teos/cli/teos_cli.py

```diff
@@ -126,6 +138,10 @@ def main(command, args, command_line_conf):
                 sys.exit(f"Expected only one argument, not {len(args)}")
 
             result = rpc_client.get_user(args[0])
+
+        elif command == "stop":
+            result = rpc_client.stop()
```

Comment:
This is not returning anything now

Reply:
Fixed in f3135e8, and initialized `result` to `None` before all the commands, just to be more explicit.